package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "link": "npm install -g",
     "unlink": "npm uninstall -g",
-    "test": "jest test/",
+    "test": "jest test/ && jest test/spectral/tests/spectral-validator.test.js --runInBand",
+    "test:seq": "jest test/spectral/tests/spectral-validator.test.js --runInBand",
```

Comment:
I added tests to check reading the default config .validaterc, and the user config .spectral.yml, but to test them I had the test create a temporary config file.

However, this interferes with other tests when run in parrallel, so I have this test file running sequentially after the regular tests.

Is there a better solution for tests like these or is it fine to run this file sequentially?

Reply:
Maybe try to spy on the fs package? https://stackoverflow.com/questions/50066138/mock-fs-function-with-jest . Not sure, @dpopp07 might have more experience doing something like that.